tests/backup/backup_test.go

```diff
@@ -547,6 +545,179 @@ var _ = Describe("{ShareBackupWithUsersAndGroups}", func() {
 	})
 })
 
+
+// This testcase verifies alternating backups between locked and unlocked bucket
+var _ = Describe("{BackupAlternatingBetweenLockedAndUnlockedBuckets}", func() {
+	var (
+		appList = Inst().AppList
+	)
+	var preRuleNameList []string
+	var postRuleNameList []string
+	var contexts []*scheduler.Context
+	labelSelectors := make(map[string]string)
+	CloudCredUIDMap := make(map[string]string)
+	BackupLocationMap := make(map[string]string)
+	var backupList []string
+	var appContexts []*scheduler.Context
+	var backupLocation string
+	var bkpNamespaces []string
+	var clusterUid string
+	var clusterStatus api.ClusterInfo_StatusInfo_Status
+	bkpNamespaces = make([]string, 0)
+	providers := getProviders()
+	JustBeforeEach(func() {
+		StartTorpedoTest("Backup: BackupAlternatingBetweenLockedAndUnlockedBucket", "Deploying backup", nil, 0)
```

Comment:
As a general rule, @lsrinivas-pure has asked us not to include special characters like `:` in the test description. And also there should not be any space in this string
`Backup: BackupAlternatingBetweenLockedAndUnlockedBucket`